### ISSUE TITLE
Components / Progress bar - color contrast

### ISSUE THREAD

**Issue by yesenia-gtz** [Office-of-Digital-Services/California-State-Web-Template-Website#977]
[Components / Progress bar](https://as-cdt-pub-migr-betaws-ww-p-001-stage.azurewebsites.net/components/progress-bar.html)

Color contrast is not sufficient. Text on this page does not meet WCAG minimum color-contrast requirements.

![Screen Shot 2022-12-13 at 1 27 53 PM](https://user-images.githubusercontent.com/98193284/207447537-e9c4a593-60e8-4089-9847-6127195c1e70.png)

- [ ] [Change Text field (60%)text contrast to: #000000](url)

**Comment by sharon-rabang** [comment]
@yesenia-gtz Hi! Did you get a different color contrast ration result (6:4:1) from this one?
![image](https://user-images.githubusercontent.com/99768807/208505490-07979041-e9b8-4dbd-85af-06a422ce3b54.png)

**Comment by yesenia-gtz** [comment]
HI!, this was part of a11y review. @pglomboy  can you clarify for @sharon-rabang ?

**Comment by pglomboy** [comment]
@sharon-rabang yes please. change the contrast to #000000 to pas minimum AAA requirements.
[Change Text field (60%)text contrast to: #000000](https://github.com/Office-of-Digital-Services/California-State-Web-Template-Website/issues/url)

**Comment by sharon-rabang** [comment]
@pglomboy Thanks for clarifying. @kkoryaka Please update the "60%" text from #3B3A48 to #000000. Thanks!

**Comment by yesenia-gtz** [comment]
@Greg-Watanabe can you please take care of this one?

Please update the "60%" text from #3B3A48 to #000000. Thanks!

**Comment by Greg-Watanabe** [comment]
@sharon-rabang @pglomboy @yesenia-gtz 
This is done
https://as-cdt-pub-migr-betaws-ww-p-001-stage.azurewebsites.net/components/progress-bar.html

**Comment by yesenia-gtz** [comment]
Thank you, closing out
